Packages/com.unity.inputsystem/Tests/InputSystem/Plugins/DualShockTests.cs

```diff
@@ -109,11 +109,93 @@ public void Devices_CanSetLightBarColorAndMotorSpeedsOnDualShockHID()
         Assert.That(receivedCommand.Value.redColor, Is.EqualTo((byte)(0.123f * 255)));
         Assert.That(receivedCommand.Value.greenColor, Is.EqualTo((byte)(0.456f * 255)));
         Assert.That(receivedCommand.Value.blueColor, Is.EqualTo((byte)(0.789f * 255)));
-    }
-
+    }
+
 #endif
-
+
 #if UNITY_EDITOR || UNITY_PS4
+
+    [Test]
+    [Category("Devices")]
+    public void Devices_CanSetLightBarColorAndMotorSpeedsOnDualShockPS4()
+    {
+        var device = InputSystem.AddDevice(new InputDeviceDescription
+        {
+            deviceClass = "PS4DualShockGamepad", ////REVIEW: this should be the product name instead
```

Comment:
Feel free to cull those comments when copypasting :) This comment isn't even quite correct :)